src/include/cursor.h

```diff
@@ -152,6 +152,8 @@ struct __wt_cursor_btree {
      *
      * A temporary buffer for building on-page keys when searching row-store
      * files.
+     *
+     * A temporary buffer for tracking random key return values to avoid repetition.
```

Comment:
This is unclear to me. Having such a similar comment right after the lines above with no fields in between makes me think the new comment is supposed to supercede the one on line 153. Maybe:
`A temporary buffer for building on-page keys when searching row-store files or for tracking random key return values.`

Reply:
Sure, fixed in 7bbad1497 (I numbered the paragraphs so it's clear there are multiple uses).